controllers/tools.js

```diff
@@ -31,6 +31,7 @@ const addToolHandler = (req, res, next) => {
 
 	api(req).get('/ltiTools', { qs: { isTemplate: true } })
 		.then((tools) => {
+			const filterTools = tools.data.filter((tool) => tool.name !== 'SchulcloudNextcloud');
```

Comment:
Is there a way to add this logic to the server request? (potentially in conjunction with a parameter or seperate route to control if the user gets all tools, or only those he can add himself...)

Are there any other tools this is already solved for? 

please at the very least define a constant array of tools to be filtered. That can later then be refactored to come from somewhere else, like the server.

Reply:
this logic will be added to the server with the tickets N21-90 and N21-91 when we do the associated refactoring tasks. Until then it is safer and less intrusive to filter in the frontend

Reply:
👍